src/serialmonitor/outputBuffer.ts

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+import { performance } from "perf_hooks";
+import * as vscode from "vscode";
+
+export class BufferedOutputChannel implements vscode.Disposable {
+    private _buffer: string[];
+    private timer: NodeJS.Timer;
```

Comment:
Nit: `_timer`. Most of the rest of the code we work on doesn't use this convention, but the Arduino codebase does.